juniper_official/System/system-storage-capacity.rule

```diff
@@ -91,6 +91,10 @@ healthbot {
             rule-properties {
                 version 2;
                 contributor juniper;
+                category advanced;
+                is-scaling-rule {
```

Comment:
System storage is not scale rule as juniper device will have hardly <10 mount paths

Reply:
Removed scaling.